FlowCrypt/Functionality/DataManager/Encrypted Storage/EncryptedStorage.swift

```diff
@@ -223,6 +224,14 @@ extension EncryptedStorage: PassPhraseStorageType {
             .where({ $0.user.email == email })
             .compactMap(PassPhrase.init)
     }
+
+    func removePassPhrases(for email: String) throws {
+        let keyPairs = try storage.objects(KeypairRealmObject.self)
+            .where({ $0.user.email == email })
+        for keyPair in keyPairs {
+            try updateKeys(with: keyPair.primaryFingerprint, passphrase: nil)
+        }
+    }
```

Comment:
Is this new method `EncryptedStorage.removePassPhrases` used? From from the PR it appears it's not

<img width="613" alt="image" src="https://user-images.githubusercontent.com/6306961/168243362-1c450097-cbc2-4716-bedb-ea7bff975e7e.png">

Reply:
Looks like it doesn't called on logout, I missed it during review.
It'll be good to add another test for checking if passphrases are deleted from `EncryptedStorage` on logout.

Reply:
EncryptedStorage.removePassPhrases is not used.

Reply:
When user logs out, encryptedStorage.logoutUser method is called and all related passphrases deleted too though.

Reply:
Pass phrases are a column on keys table. So they should, as long as keys are.

Reply:
When encryptedStorage.logoutUser method is called all user related objects get deleted as well as KeyPair & pass phrases.
![image](https://user-images.githubusercontent.com/77344191/168248823-5dd03505-84aa-4c83-aea8-08fec7a6d7af.png)

Reply:
> EncryptedStorage.removePassPhrases is not used. removePassPhrases function needed to be decleared in PassPhraseStorageType and I added EncryptedStorage.removePassPhrases just in case.

YAGNI principle - let's remove the newly added EncryptedStorage.removePassPhrases in #1607 . Thanks!

Reply:
But EncryptedStorage inherits from PassPhraseStorageType protocol and PassPhraseStorageType has removePassPhrases method.

Reply:
Do you wanna make new protocol which only has `removePassPhrases` function and let InMemoryPassPhraseStorage inherits that protocol?

Reply:
@tomholub

Reply:
Ok - now I understand. Let's keep it then, no issue.

Reply:
ok